msvs/testmf6.sln

```diff
@@ -0,0 +1,31 @@
+﻿
```

Comment:
should this be moved to a different directory or subdirectory for better organization?

Reply:
Ah, well spotted, I believe that's a remnant from when I thought that unit testing through such a VS project could be a good long term strategy for MODFLOW. Now, I believe our way of working with the autotests, and the API for more invasive tests, is to be preferred. I will delete this solution file.